lib/faker/date.ex

```diff
@@ -51,9 +51,12 @@ if Version.match?(System.version(), ">= 1.3.0") do
         DateTime.utc_now()
         |> DateTime.to_unix()
 
-      sign = if days < 0, do: -1, else: 1
-
-      unix_now + sign * @seconds_per_day * :crypto.rand_uniform(1, abs(days))
+      days
+      |> Kernel.<(0)
+      |> if(do: -1, else: 1)
+      |> Kernel.*(@seconds_per_day)
+      |> Kernel.*(:crypto.rand_uniform(1, abs(days)))
+      |> Kernel.+(unix_now)
       |> DateTime.from_unix!()
       |> DateTime.to_date()
     end
```

Comment:
I agree. I honestly didn't expect this series of pipes to make it through, it was fun though.